src/components/SearchOverlay/SearchOverlay.js

```diff
@@ -357,6 +365,9 @@ class SearchOverlay extends React.PureComponent {
             <Input id="case-sensitive-option" type="checkbox" ref={this.caseSensitiveInput} onChange={this.onChangeCaseSensitive} label={t('option.searchPanel.caseSensitive')} />
             <Input id="whole-word-option" type="checkbox" ref={this.wholeWordInput} onChange={this.onChangeWholeWord} label={t('option.searchPanel.wholeWordOnly')} />
           </div>
+          <div className={`wildcard ${isSearchPanelOpen ? 'visible' : ''}`}>
+              <Input id="wild-card-option" type="checkbox" ref={this.wildcardInput} onChange={this.onChangeWildcard} label={t('option.searchPanel.wildcard')} />
```

Comment:
Hmm it'd be nice if we can move it to the div above.
Could you try playing around the `<div className="options ...">` div to see if you can use flex-wrap to put these three checkboxes into one div?

Reply:
Also the indentation should be 2 instead of 4.

Reply:
Looks good! One more thing, can you also add a line below 83 so that when we programmatically search the text with wildcard option being true, the checkbox will be checked?

You can verify if it's working by:
```javascript
instance.searchTextFull('You', { wildcard: true })
```

or switch to the iframe using the dev tool and then in the console
```javascript
readerControl.searchTextFull('You', { wildcard: true })
```

Reply:
Made changes as requested, thanks!